Mixpanel/Mixpanel.m

```diff
@@ -696,6 +722,7 @@ - (void)archiveProperties
     [p setValue:self.people.unidentifiedQueue forKey:@"peopleUnidentifiedQueue"];
     [p setValue:self.shownNotifications forKey:@"shownNotifications"];
     [p setValue:self.timedEvents forKey:@"timedEvents"];
+    [p setValue:self.automaticEventsEnabled forKey:@"automaticEvents"];
```

Comment:
since we have to handle that case for the first time i think it's fine since it does not add any extra complexity. worried about users seeing pre-track events days after they turned it off.